File: AULA_02/C/PedraPapelTesoura.c
#include <stdio.h>

int main(){
  int jogador1, jogador2;

  printf("Pedra (0), Papel (1), Tesoura (2): ");
  scanf("%d",&jogador1);
  printf("Pedra (0), Papel (1), Tesoura (2): ");
  scanf("%d",&jogador2);

  if( jogador1 > jogador2 || jogador1==0 && jogador2 == 2 ){
    printf("Jogador1 venceu\n");
  }else if( jogador2 > jogador1 || jogador2 == 0 && jogador1 == 2 ){
    printf("Jogador2 venceu\n");
  }else{
    printf("Empate\n");
  }

  return 0;

}

File: AULA_02/Haskell/PedraPapelTesoura.hs
vencedor a b
      | a>b || (a==0 && b==2) = "Jogador1 venceu"
      | b>a || (a==2 && b==0) = "Jogador2 venceu"
      | otherwise = "Empate"

main = do
       putStrLn "Pedra (0), Papel (1), Tesoura (2)"
       line <- getLine
       let jogador1 = read line

       putStrLn "Pedra (0), Papel (1), Tesoura (2)"
       line <- getLine
       let jogador2 = read line

       putStrLn $ vencedor jogador1 jogador2
